recipes/default.rb

```diff
@@ -43,7 +43,8 @@
   'rate-limiting'         => node['repose']['rate_limiting'        ]['cluster_id'],
   'slf4j-http-logging'    => node['repose']['slf4j_http_logging'   ]['cluster_id'],
   'translation'           => node['repose']['translation'          ]['cluster_id'],
-  'uri-identity'          => node['repose']['uri_identity'         ]['cluster_id']
+  'uri-identity'          => node['repose']['uri_identity'         ]['cluster_id'],
+  'api-validator'         => node['repose']['api_validator'        ]['cluster_id']
```

Comment:
Super nit pick, but the rest of these are alphabetical. Would you mind moving this up the list?